### DISCUSSION TITLE
Etched compatability

### DISCUSSION THREAD

**Discussion by axtrough** [P3pp3rF1y/SophisticatedBackpacks#1202] [Ideas]
It would be nice if the custom discs from etched would work with the jukebox upgrade.

**Comment by P3pp3rF1y** [comment]
This was suggested a few times, would require compatibility in pre 1.21 versions. From 1.21 on there's a way to add custom music with just standard components so if Etched actually continues they should use this and it will be automatically supported there.

**Reply by axtrough** [reply]
You mean that in 1.21 version of sophisticated backpacks it would have support if etched updated to 1.21?

**Reply by P3pp3rF1y** [reply]
Yes, unless I am missing anything they should be able to use the vanilla system and so jukebox upgrade should be able to play their discs by default.

**Comment by AAMBIENTT** [comment]
still waiting on this

**Reply by AAMBIENTT** [reply]
sounded mean sorry!
